src/main/java/org/apache/sling/distribution/journal/bookkeeper/ContentPackageExtractor.java

```diff
@@ -47,15 +47,15 @@
  * Each distribution package is inspected for possible content packages in /etc/packages.
  * Such content packages are installed via the Packaging service.
  */
-class ContentPackageExtractor {
+public class ContentPackageExtractor {
```

Comment:
maybe move this class to the subscriber package and keep it package scoped instead?

Reply:
It needs to be public as it is used in `PackageHandlerFactory` and its implementation, where the EDS one (`EdsPackageHandlerFactory`) will be in sites-franklin.